app/src/queries/astroport/useAssetPriceInUstQuery.ts

```diff
@@ -40,11 +40,9 @@ const assetPriceQuery = async (
   const assetPoolSize = assetShare.amount;
   const ustPoolSize = ustShare.amount;
 
-  const assetPrice = big(ustPoolSize)
+  return big(ustPoolSize)
     .div(+assetPoolSize === 0 ? '1' : assetPoolSize)
```

Comment:
prefer the use of ```Big(assetPoolSize)``` instead of using the "+" as the plus is just used as an implicit type conversion.

Reply:
✅ https://github.com/Anchor-Protocol/anchor-web-app/pull/317/commits/0f027e840585c0623faae284baddf18b3f9120a8